src/python/gem5/components/boards/se_binary_workload.py

```diff
@@ -109,29 +241,17 @@ def set_se_binary_workload(
         # SE-mode simulation.
         self._set_fullsystem(False)
 
-        binary_path = binary.get_local_path()
-        self.workload = SEWorkload.init_compatible(binary_path)
-
-        process = Process()
-        process.executable = binary_path
-        process.cmd = [binary_path] + arguments
-        if stdin_file is not None:
-            process.input = stdin_file.get_local_path()
-        if stdout_file is not None:
-            process.output = stdout_file.as_posix()
-        if stderr_file is not None:
-            process.errout = stderr_file.as_posix()
-        if env_list is not None:
-            process.env = env_list
+        process = self._create_process(
+            binary=binary,
+            pid=100,
+            stdin_file=stdin_file,
+            stdout_file=stdout_file,
+            stderr_file=stderr_file,
+            env_list=env_list,
+        )
```

Comment:
```suggestion
        process = self._create_process(
            binary=binary,
            pid=100,
            stdin_file=stdin_file,
            stdout_file=stdout_file,
            stderr_file=stderr_file,
            env_list=env_list,
            arguments=arguments,
        )
```

The parameter `arguments` should also be passed otherwise we wont we able to run binaries that require arguments.

Reply:
Oh, I missed the parameter. Thanks for pointing it out.